usaspending_api/common/serializers.py

```diff
@@ -7,83 +7,127 @@ class LimitableSerializer(serializers.ModelSerializer):
-
-            if params.get('verbose', False):
-                # We have a request for verbose, so we return here so that we
-                # return all fields
-                return
-
-            # We must exclude before include to avoid conflicts from user error
-            if exclude_fields is not None and not self.context.get("child", False):  # the child check should be removed when child field limiting is implemented
-                for field_name in exclude_fields:
-                    try:
-                        self.fields.pop(field_name)
-                    except KeyError:
-                        # Because we're not currently handling nested serializer field
-                        # limiting pass-down, this can happen due to the context pass down
-                        pass
-
-            if include_fields is not None and not self.context.get("child", False):  # the child check should be removed when child field limiting is implemented
-                allowed = set(include_fields)
-                existing = set(self.fields.keys())
-                for field_name in existing - allowed:
-                    try:
-                        # If we have a coded field, always include its description
-                        if field_name.split("_")[-1] == "description" and "_".join(field_name.split("_")[:-1]) not in allowed:
-                            continue
-                        self.fields.pop(field_name)
-                    except KeyError:
-                        # Because we're not currently handling nested serializer field
-                        # limiting pass-down, this can happen due to the context pass down
-                        pass
-
-            elif current_viewset and current_viewset.action == 'retrieve':
-                # The view has specifically asked that all fields should
-                # be returned (for example, when the request url follows
-                # the item/{pk} pattern)
-                return
-
-            else:
-                try:
-                    include_fields = self.Meta.model.get_default_fields(path=request._request.path_info)
-                    allowed = set(include_fields)
-                    existing = set(self.fields.keys())
-                    for field_name in existing - allowed:
-                        self.fields.pop(field_name)
-                except AttributeError:
-                    # We don't have get default fields available
-                    pass
+        # Now we alter our own field sets
+        # We must exclude before include to avoid conflicts from user error
+        for field_name in exclude_fields:
+            self.fields.pop(field_name)
+
+        if len(include_fields) > 0:
+            allowed = set(include_fields)
+            existing = set(self.fields.keys())
+            for field_name in existing - allowed:
+                # If we have a coded field, always include its description
```

Comment:
I can't remember if we've always had this feature of automatically including the description of coded fields (I think it's been in since the story to add all of those `_description` fields?). 

If it's existing logic we should leave it in, but maybe a consideration for API 2.0 is am thinking more about the trade-offs between a convenience vs coupling the serializer to specific model names and deviating from our standard pattern of explicitly requesting fields when they're needed.

I know it's more nuanced here because these fields aren't always (ever?) in the DAIMS, so harder to surface them.